### PR TITLE
Simplify override description

### PR DESCRIPTION
**Pull request by TheYoBots**

### CODE REVIEW

**Comment by TheYoBots** [comment]
https://github.com/TheYoBots/lichess-bot/blob/simplify-overrides/wiki/Configure-lichess-bot.md#challenging-other-bots

**Review by MarkZH — APPROVED** [review]

**Comment by TheYoBots** [comment]
Before this is merged, I think the wiki `.md` file can also be simplified further. It still is slightly confusing.

**Comment by TheYoBots** [comment]
It's slightly improved now. Do feel free to check if there is anything else that I need to change.

File: config.yml.default
Comment (AttackingOrDefending) [inline config.yml.default:230]: Add a new line at the end of the file.

**Review by AttackingOrDefending — APPROVED** [review]

**Comment by AttackingOrDefending** [comment]
@MarkZH I won't merge this yet, because changes have been done since you approved it. If you approve the new changes, go ahead and merge this.

File: wiki/Configure-lichess-bot.md
Comment (MarkZH) [inline wiki/Configure-lichess-bot.md:243]: Replace the `<br/>` with an empty line below this one.

File: wiki/Configure-lichess-bot.md
Comment (MarkZH) [inline wiki/Configure-lichess-bot.md:248]: Replace the `<br/>` with an empty line below this one.

Comment (TheYoBots) [reply wiki/Configure-lichess-bot.md:248]: Override section using newline:
image
Override section using `<br/>`:
image
This is why I used `<br/>`.

Comment (MarkZH) [reply wiki/Configure-lichess-bot.md:248]: I see. These should still be paragraph breaks with extra space, so how about adding a second `<br/>` on the same line in both places?

**Review by MarkZH — APPROVED** [review]

**Pull request merged**
